Presentation/Alert.swift

```diff
@@ -90,24 +90,28 @@ public extension PresentationStyle {
                 fatalError("presented view controller must be an UIAlertController to be presented as a action sheet")
             }
 
-            var disposer: Disposable
+            let bag = DisposeBag()
             if let createSheet: () -> (UIAlertController, Disposable) = vc.associatedValue(forKey: &createSheetKey) {
+                var disposer: Disposable
                 (vc, disposer) = createSheet()
-            } else {
-                disposer = NilDisposer()
+                bag += disposer
             }
 
             if let presenter = vc.popoverPresentationController {
-                guard let view = sourceView else {
-                    fatalError("Sheet style requires a from view if presented in popover")
+                presenter.sourceView = sourceView ?? from.view
+
+                if let rect = sourceRect {
```

Comment:
Thanks for the great fix. From an outside point of view I have the feeling that the knowledge why we take the center as a source rect here can get lost pretty easily. Does it make sense to add a comment why we introduced this and maybe what the problem with the normal bounds was?

Reply:
you are right, I will open a new pr with a comment